data/sql_setup/prepare_schedule_d.sql

```diff
@@ -0,0 +1,19 @@
+-- Create trigger to maintain Schedule C search columns
+ALTER TABLE disclosure.nml_sched_d DROP COLUMN if EXISTS creditor_debtor_name_text;
+
+ALTER TABLE disclosure.nml_sched_d add column creditor_debtor_name_text tsvector;
```

Comment:
Is this safe to do, is there no data in it before dropping and renaming the column?  And are we doing this because the column is an incorrect data type?  If so, would a single statement like this work?

```sql
ALTER TABLE disclosure.nml_sched_d ALTER COLUMN creditor_debtor_name_text TYPE tsvector;
```